5.5/examples/replica/mysql_replica.json

```diff
@@ -68,6 +68,23 @@
       }
     },
     {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "mysql-slave"
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce"
+        ],
+        "resources": {
+          "requests": {
+            "storage": "512Mi"
```

Comment:
This needs to be a parameter with a default of 1gig because online doesn't have 512meg volumes, see:
https://github.com/openshift/origin/blob/master/examples/db-templates/mongodb-persistent-template.json#L232-L237
https://github.com/openshift/origin/blob/master/examples/db-templates/mongodb-persistent-template.json#L43-L57